<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenFileOrProjectDialog</class>
 <widget class="QDialog" name="OpenFileOrProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Open File or Project</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QCommandLinkButton" name="openFileButton">
     <property name="text">
      <string>Open File</string>
     </property>
     <property name="description">
      <string>Open a file</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCommandLinkButton" name="openProjectButton">
     <property name="text">
      <string>Open Project</string>
     </property>
     <property name="description">
      <string>Open a project folder</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCommandLinkButton" name="openProjectFileButton">
     <property name="text">
      <string>Other - Open Project File Based Project</string>
     </property>
     <property name="description">
      <string>Open a project file based project</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OpenFileOrProjectDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OpenFileOrProjectDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
